HARK/ConsumptionSaving/ConsRiskyContribModel.py

```diff
@@ -1206,7 +1206,7 @@ def post_return_derivs(inc_shocks, b_aux, g_aux, s):
 
         # Define grids
         b_aux_grid = np.concatenate([np.array([0.0]), Rfree * aXtraGrid])
-        g_aux_grid = np.concatenate([np.array([0.0]), max(RiskyDstn.X) * nNrmGrid])
+        g_aux_grid = np.concatenate([np.array([0.0]), max(RiskyDstn.X.flatten()) * nNrmGrid])
```

Comment:
It seems like a lot of extra code is now needed to flip vertical vectors/arrays into horizontal ones.
Would it be possible to have the arrays oriented horizontal in the DiscreteDistribution object?

Reply:
Possible yes, but I still thought this would be the path of least resistance.

I think other parts of HARK that deal with multivariate distributions use things like `dstn.X[n]` to find all the possible draws of the nth dimension. This seems like an intuitive and compact code pattern that I would like to keep. It was easier for me to conceptually think that nature was always the last dimension.

Reply:
"X" is truly a terrible name for an object property that is doing so much work. See #1051 

I'm not suggesting you replace "X" with something else in this PR, but I wonder what mathematical object X is now.

It's possible that we could have multiple ways of accessing this data within a distribution, so it requires less manipulation before entering and after pulling it from the object.